lambdas/build-stac/utils/events.py

```diff
@@ -15,15 +16,19 @@ class BaseEvent(BaseModel, frozen=True):
     asset_roles: Optional[List[str]] = None
     asset_media_type: Optional[Union[str, pystac.MediaType]] = None
 
+    start_datetime: Optional[datetime] = None
+    end_datetime: Optional[datetime] = None
+    single_datetime: Optional[datetime] = None
```

Comment:
Should this go on the `BaseEvent` or the `RegexEvent`?  It seems that only the `RegexEvent` makes use of this value.  Is it actually possible to get a `CmrEvent` with any of these values set?

Reply:
you're right, it should probably go to regexevent